src/litdata/streaming/cache.py

```diff
@@ -47,6 +48,10 @@ def __init__(
         writer_chunk_index: Optional[int] = None,
         storage_options: Optional[Dict] = {},
         max_pre_download: int = 2,
+        epoch: Optional[int] = None,
+        config: Optional[ChunksConfig] = None,
+        on_start_pre_item_download_count: int = 100,
```

Comment:
I think this was a good idea to try this out but I think we should just pass download_bytes_size="1MB"